src/core_ocean/mode_init/mpas_ocn_init_hurricane.F

```diff
@@ -324,16 +324,8 @@ subroutine ocn_init_setup_hurricane(domain, iErr)!{{{
             end if
           end do
 
-          ! shift coordinates to accomodate wetting and drying
-          do k = 1, nVertLevels
-            refBottomDepth(k) = refBottomDepth(k) - globalMinBottomDepth
-          end do
-
           do iCell = 1, nCellsSolve
 
-            bottomDepth(iCell) = bottomDepth(iCell) - globalMinBottomDepth
-            ssh(iCell) = ssh(iCell) + globalMinBottomDepth
-
```

Comment:
@mark-petersen, if you can approve this merge into the MPAS-O fortran code, we can merge this in after @sbrus89 verifies it compiles and produces reasonable results for him.

Reply:
There are only about 6 lines and they remove changes needed to account for the fix you did ensuring we can run with layers above mean sea level.